File: BOJ/25000 ~ 26000/P25314.java
// https://www.acmicpc.net/problem/25314
import java.util.Scanner;

public class P25314 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int numOfLong = N / 4;

        StringBuilder sb = new StringBuilder();
        for (int i= 0; i < numOfLong; i++) {
            if (sb.length() == 0) {
                sb.append("long");
            } else {
                sb.append(" long");
            }
        }

        sb.append(" int");
        System.out.println(sb);
    }
}

File: BOJ/25000 ~ 26000/P25314.kt
import java.util.Scanner

fun main(args: Array<String>) = with(Scanner(System.`in`)) {
    val n = nextInt()
    val numOfLong = n / 4

    val sb = StringBuilder()

    IntRange(1, numOfLong).forEach { _ ->
        if(sb.isEmpty()) sb.append("long")
        else sb.append(" long")
    }

    sb.append(" int")
    print(sb)
}
